package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "archethic",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Just can you remove the versioning, we will change the version later, during the release

Reply:
done